InsiteTeamTask/Controllers/AttendanceController.cs

```diff
@@ -1,23 +1,59 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using InsiteTeamTask.Models;
-using InsiteTeamTask.Repositories;
+using InsiteTeamTask.Services;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
 
 namespace InsiteTeamTask.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class AttendanceController : ControllerBase
     {
+        private readonly IAttendenceViewService _attendenceViewService;
+        private readonly ILogger<AttendanceController> _logger;
+
+        public AttendanceController(IAttendenceViewService attendenceViewService, ILogger<AttendanceController> logger)
+        {
+            _attendenceViewService = attendenceViewService;
+            _logger = logger;
+        }
+
         // GET api/values
-        [HttpGet]
-        public ActionResult<IEnumerable<Attendance>> Get()
+        [HttpGet()]
+        [Route("get/{gameNumber}/{seasonNumber}")]
```

Comment:
Whilst I like the demonstration of API routing here, RESTful API routes should work in such a way that adding values to the URL chain should convey a hierarchy of those objects.

In this case you're insinuating that Attendance has a child "Game", which in turn has a child "Season". Since neither of these things are true, I'd prefer to see the following:
```
[HttpGet]
public IEnumerable<Attendance> Get([FromQuery]int gameId, [FromQuery]int seasonId)
```
This relates to a route: https://appUrl/api/attendance?gameId=3&seasonId=1